k9mail/src/main/java/com/fsck/k9/view/MessageHeader.java

```diff
@@ -154,10 +185,31 @@ private void onAddSenderToContacts() {
         }
     }
 
+    private void onAddAddressesToClipboard(Address[] addresses){
+        if(mMessage != null){
+            try{
+                String addressesToCopy = "";
```

Comment:
You might want to use `Address.toString(Address[])` here. It returns a comma separated list of addresses.

If you want to continue using a space separated list please use a `StringBuilder` to construct that string.